lib/Backend/Peeps.cpp

```diff
@@ -303,7 +303,7 @@ Peeps::PeepAssign(IR::Instr *assign)
     IR::Instr *instrNext = assign->m_next;
 
     // MOV reg, sym
-    if (src->IsSymOpnd())
+    if (src->IsSymOpnd() && src->AsSymOpnd()->m_offset == 0)
```

Comment:
A sym operand can refer to different offset of a stack sym.   The peep can only track one offset because the reg is store on the stack sym itself.  So we don't only peep if the sym operand referencing the beginning of the stack sym.